src/nnfusion/engine/pass/graph/autodiff_pass.hpp

```diff
@@ -33,7 +33,8 @@ namespace nnfusion
             class AutodiffPass : public GraphPassBase
             {
             public:
-                bool run_on_graph(std::shared_ptr<nnfusion::graph::Graph>& graph) override;
+                bool run_on_graph(std::shared_ptr<nnfusion::graph::Graph>& graph,
+                                  std::shared_ptr<vector<vector<float>>> backward_inputs);
```

Comment:
❔ [Question]

So you are creating a new function instead of override the original one? Is there any implementation of the original function in base class, and is that function run as you expect?

Reply:
Change to keep the override function.